include/networkit/centrality/TopCloseness.hpp

```diff
@@ -78,6 +81,7 @@ class TopCloseness final : public Algorithm {
     count k;
     bool first_heu, sec_heu;
     std::vector<node> topk;
+    std::vector<node> nodeList;
```

Comment:
Can we avoid to copy the node list?
```suggestion
    const std::vector<node> &nodeList;
```

Reply:
Changed it to be a pointer due to removing it from the constructor parameter list.